examples/etcd/resources/monitor-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Definition
+metadata:
+  name: monitor-clustername-job
+meta:
+  timeout: 0
+job:
+  apiVersion: batch/v1
+  kind: Job
```

Comment:
About second bullet - deleting replicaset via kubectl deletes underlying pods, the `--cascade` flag is on by default.

I will approve these changes, but maybe we can think about having the loop in the k8s instead of in container? Like returning an error from the monitoring command to restart the pod, but this also seems really hacky and not such a good idea :D.